dynawo/sources/Simulation/DYNSimulation.h

```diff
@@ -212,13 +213,23 @@ class Simulation {
   inline void setTimelineOutputFile(const std::string& outputFile) {
     timelineOutputFile_ = outputFile;
   }
+
   /**
    * @brief setter for the export mode of the timeline
-   * @param mode timeline's mode export
+   * @param mode timeline's export mode
    */
   inline void setTimelineExportMode(const exportTimelineMode_t& mode) {
     exportTimelineMode_ = mode;
   }
+
+  /**
+   * @brief getter for the export mode of timeline
+   * @return timeline's export mode
+   */
+  inline const exportTimelineMode_t& getTimelineExportMode() const {
```

Comment:
no need of const & for an enum (same above)

Reply:
of course... But looking at an object exportTimelineMode_t one does not know if it is an enum or not. If it turns to be something else later ( an enum class or even a const object ) that code will fit. I don't think I used that getter anyway. It is because it should always be like that...

Reply:
If not used, please remove the method, otherwise please remove the reference as it should not be used with fundamental data types (and if we change it to a class then this should be updated)